dashboard/src/main/resources/webapp/static/lib/angular-topology-editor/credentials.js

```diff
@@ -253,14 +253,19 @@ var Credentials = (function() {
      * Returns a topology from a DAM.
      */
     function to_topology(rawdam) {
-        var PAAS_TYPE_PREFIX = "seaclouds.nodes.Platform";
-        var IAAS_TYPE_PREFIX = "seaclouds.nodes.Compute";
-        var INSTANCES = "instancesPOC";
+        var cloud_prefixes = [
+                "seaclouds.nodes.Platform",
+                "tosca.nodes.Platform",
+                "seaclouds.nodes.Compute",
+                "tosca.nodes.Compute"
+        ];
```

Comment:
Yes. We can consider this PR as an intermediate step before that removal. After it, I can submit another PR to delete seaclouds.nodes.XXX from the array.